src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -118,12 +109,51 @@ function addRubricCol(questionNum) {
         $(rubricRowFragment).insertAfter(lastTd);
     }
 
+    // Add options row at the end
+    const optionsRow =
+        `<td class="align-center rubricCol-${questionNum}-${newColNumber - 1}" data-col="${newColNumber - 1}">
+            <div class="btn-group">
+                <button type="button" class="btn btn-default" id="rubric-move-col-left-${questionNum}-${newColNumber - 1}"
+                        onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, true)" data-toggle="tooltip"
+                        data-placement="top" title="Move column left">
+                    <span class="glyphicon glyphicon-arrow-left"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubricRemoveChoiceLink-${questionNum}-${newColNumber - 1}"
+                        onclick="removeRubricCol(${newColNumber - 1}, ${questionNum})"
+                        onmouseover="highlightRubricCol(${newColNumber - 1}, ${questionNum}, true)"
+                        onmouseout="highlightRubricCol(${newColNumber - 1}, ${questionNum}, false)">
+                    <span class="glyphicon glyphicon-remove"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubric-move-col-right-${questionNum}-${newColNumber - 1}"
+                        onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, false)" data-toggle="tooltip"
```

Comment:
Sorry, I already added the `onclick` handlers. This is old code, forgot to remove. When any change is made in HTML, I need to update the `.html` file in resource as well as the code here. So I keep forgetting it :sweat_smile: